src/utilities/path.ts

```diff
@@ -1,3 +1,4 @@
+/** @hidden */
```

Comment:
I think this can be combined with the lines below to make one comment block

Reply:
Done.